packages/backend/src/peripherals/database/ReportRepository.ts

```diff
@@ -63,25 +63,29 @@ export class ReportRepository extends BaseRepository {
     return rows.map(toRecord)
   }
 
-  async addOrUpdateMany(reports: ReportRecord[]) {
+  async addOrUpdateMany(
+    reports: ReportRecord[],
+    where: { timestamp: UnixTime; chainId: ChainId },
+  ) {
+    if (reports.length === 0) {
+      return 0
```

Comment:
we still need to remove records from the database in this case

Reply:
https://github.com/l2beat/l2beat/pull/1838/commits/65b872dba0346c18c980447d4e516c3ebbd9f1ed